guides/get-started-microservices-on-kubernetes/simple-microservice-part1.adoc

```diff
@@ -0,0 +1,252 @@
+= {simple-microservice-part1}
+:summary: ONELINE EXPLANATION OF THE GUIDE
+:includedir: ../_includes
+include::{includedir}/_attributes.adoc[]
+include::./_includes/_titles.adoc[]
+:prerequisites-time: 10
+
+In this guide, you will learn how to create and run a link:https://docs.docker.com/guides/docker-concepts/the-basics/what-is-an-image/[Docker Image] containing a link:https://jakarta.ee/specifications/restful-ws/[Jakarta RESTful Web Service] implemented using WildFly.
+
+include::{includedir}/_prerequisites.adoc[]
+
+* link:/get-started[Getting Started with WildFly]
+
+include::_includes/_constants.adoc[]
+
+== Maven Project
+
+In link:/get-started[Getting Started with WildFly], we created a simple Jakarta EE application Maven Project containing a link:https://jakarta.ee/specifications/restful-ws/[Jakarta RESTful Web Service]: we'll pick up from there;
+
+=== pom.xml
+
+Since we are going to deploy on link:https://kubernetes.io/[Kubernetes], we need to add some features that are necessary when running on link:https://kubernetes.io/[Kubernetes]: link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/[Liveness, Readiness and Startup Probes];
```

Comment:
I think it could be mentioned here that Kubernetes will be needed in the 2.part of the gude and it's not necessary to install it at this point.

Reply:
@mnovak1 thank you: I changed the wording and it should be clearer now: let me know what you think ...